lib/fastlane/plugin/changelog/actions/emojify_changelog.rb

```diff
@@ -49,6 +49,10 @@ def self.run(params)
             emojified_content.concat(emojified_string)
           else
             # Output updated line
+            # Replace '-' with '*'
+            if line.start_with?('-')
```

Comment:
Yes!

Reply:
This looks great, our team was just lamenting that the slack posts looked great but could look better.